lib/src/tile_map_json_parser.dart

```diff
@@ -0,0 +1,1279 @@
+part of tiled;
+
+class TileMapJsonParser {
+  TileMapJsonParser();
+
+  MapJson parse(String string) {
+    // TODO Check Filetype?
+    // TODO Custom parser?
+    // TODO ?.toDouble()
+    final MapJson mapJson = MapJson.fromJson(jsonDecode(string));
+    return mapJson;
+  }
+}
+
+class MapJson {
```

Comment:
I have done all the json things in one file.
Do you want it in a directory and every class in separate files?

Reply:
It is changed in separate files